### ISSUE TITLE
Markdown doesn't seem to be getting rendered correctly.

### ISSUE THREAD

**Issue by jasonlfunk** [spatie/ray#294]
**Describe the bug**

Here is my code:
```
ray($action);
ray($action['message']);
```

Here is the output:
![image](https://user-images.githubusercontent.com/685188/107369978-73f58700-6af3-11eb-8daf-eff048f4c954.png)

**Versions**
- Ray version: Version 1.12.0 (1.12.0)
- spatie/ray package version: 1.19.3
- spatie/laravel-ray package version (if applicable): 1.12.3

PHP version: PHP 7.4.14
Laravel version (if applicable): 7.30.4

**Expected behavior**
I expect the markdown output to be styled correctly. It appears to be trying to remove newlines and format it as html. But even then, only the first newline is being replaced with a `<br>` tag.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version: 11.1

**Comment by patinthehat** [comment]
@jasonlfunk Have you tried using `ray()->markdown($action['message'])`?

**Comment by jasonlfunk** [comment]
@patinthehat Nope! I didn't know that method exists. That did seem to help in this case, thanks. 

There does seem to be something wrong with the second call to ray() here though. The output doesn't really make sense - why is there 1 `<br/>` tag?

```
        ray([$body]);
        ray($body);
        ray()->markdown($body);
```

![image](https://user-images.githubusercontent.com/685188/107372110-1dd61300-6af6-11eb-9cb1-2e3d12c0664f.png)

Thanks for ray by the way. It's the biggest boost to my dev productivity that I've had in a long time.

**Comment by patinthehat** [comment]
Not sure on the `<br />`, I'd have to dig into the code to figure that one out.  If it becomes an issue in the future, please open a separate issue for it.

Ray is the work of @freekmurze, [Spatie & team](https://spatie.be).  But I agree, they definitely deserve a hearty thanks for creating it. :tada:  _(I did write that `markdown()` method, though :slightly_smiling_face:)_

**Comment by freekmurze** [comment]
@patinthehat 🙌

closing this as the original issue seems to have been solved by using the `markdown` method.

**Comment by jasonlfunk** [comment]
@freekmurze Yes, using the markdown method works fine. However, there was still a bug in regards to the way that Ray was formatting it without using the markdown method - having that "<br/>" tag get added.

It's easy enough to make happen:

`ray("*Test*\nHello\nWorld!");`

![image](https://user-images.githubusercontent.com/685188/109287435-9b0ec100-7834-11eb-93a3-9169c46f00fa.png)

Do I really need to open a separate issue for this?

**Comment by patinthehat** [comment]
I can confirm that this is related to the App and not to the `ray` package specifically.

PHP Code:

```php
ray()->send("*Test*\n\nHello\nWorld!\n\n");
```
JSON sent to Ray:
```json
{
  "uuid": "ccdaf2c5-4110-26e0-812d-01a23d098322",
  "payloads": [
    {
      "type": "log",
      "content": {
        "values": [
          "*Test*\n\nHello\nWorld!\n\n"
        ]
      },
      "origin": {
        "file": "./test-php-project-1/test.php",
        "line_number": 5
      }
    }
  ],
  "meta": {
    "php_version": "7.4.15",
    "php_version_id": 70415,
    "ray_package_version": "1.20.1.0"
  }
}
```

**Comment by AdrianMrn** [comment]
I removed the part in the app where we're replacing `\n` with `<br />`. This change will be released in 1.14.7, please check it and let me know if this is a suitable solution for your issue.
